core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/CommonLoadBalancer.scala

```diff
@@ -266,17 +306,31 @@ abstract class CommonLoadBalancer(config: WhiskConfig,
         // is important to pass to the invokerPool because they are used to determine if the invoker can be considered
         // healthy again.
         logging.info(this, s"received completion ack for health action on $invoker")(tid)
+
+        MetricEmitter.emitCounterMetric(
+          LoggingMarkers.LOADBALANCER_COMPLETION_ACK(controllerInstance, healthcheckCompletionAck))
+
         // guard this
         invokerPool ! InvocationFinishedMessage(invoker, invocationResult)
       case None if !forced =>
-        // Received an active-ack that has already been taken out of the state because of a timeout (forced active-ack).
+        // Received a completion ack that has already been taken out of the state because of a timeout (forced ack).
         // The result is ignored because a timeout has already been reported to the invokerPool per the force.
-        logging.debug(this, s"received completion ack for '$aid' which has no entry")(tid)
+        // Logging this condition as a warning because the invoker processed the activation and sent a completion
+        // message - but not in time.
+        logging.warn(
+          this,
+          s"received completion ack for '$aid' from ${invoker} which has no entry, system error=${isSystemError}")(tid)
```

Comment:
Would it be possible to compute the time delta with which this ack missed the forced ack deadline. May be logging the `start` time from `TransactionId` (not sure how that works with sequence. Probably there it may mislead)

Reply:
I checked that part... Apparently, this is the code path where the activation entry has already been removed. So we cannot easily provide this info via the activation entry.

Chetan is right with sequences. The transaction starts with the sequence and all but the first action in the sequence will have later start times than the start of the transaction. For this reason, the `tid` cannot be used easily to compute the time delta.

I suggest to better leave out this info for now instead of provide improper information that could be misleading. Even for primitive actions, computing the time delta from the start of the transaction can be improper...